src/main/java/Duke.java

```diff
@@ -1,10 +1,76 @@
+
+import java.util.Scanner;
+
 public class Duke {
+
+    private static Task[] list = new Task[100];
+    private static int index = 0;
+
+    public static void printList(Task[] list) {
+        for(int i = 0; i < index; i++) {
+            System.out.print(String.valueOf(i+1) + ". ");
+            System.out.println(list[i]);
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println("____________________________________________________________\n" +
+                        " Hello! I'm Duke :)\n" +
+                        " What can I do for you?\n" +
+                "____________________________________________________________");
+
+        Scanner scanner = new Scanner(System.in);
+        String inputString = "";
+        while(!inputString.equals("bye bye")) {
+            inputString = scanner.nextLine();
+            //Task new_task = new Task(inputString);
+            Task new_task;
+
+            if(inputString.equals("list")) {
+                printList(list);
+            } else if(inputString.contains("done")) {
```

Comment:
For each of the conditions, contents could be put in separate methods instead of a long conditional statement. Would help to improve readability.